build/config/features.gni

```diff
@@ -172,8 +172,9 @@ enable_chromevox_next = false
 # Use brlapi from brltty for braille display support.
 use_brlapi = is_chromeos
 
-# Enable WebVR support by default on Android
-# Still requires command line flag to access API
-enable_webvr = is_android
+declare_args() {
```

Comment:
You could have at least reused the same `declare_args` block that's in the beginning of the file.

Anyway, this is exactly the kind of thing you should attempt to send upstream: https://codereview.chromium.org/2198003002/